apps/web/src/pages/Pools/ManageAccessPage/ManageAccessPage.tsx

```diff
@@ -0,0 +1,194 @@
+      cell: ({ row: { original: member } }) =>
+        emailLinkCell(member.email, intl),
+    }),
+    columnHelper.accessor((member) => roleAccessor(member.roles, intl), {
+      id: "roles",
+      header: intl.formatMessage({
+        defaultMessage: "Membership roles",
+        id: "9s7Ctc",
+        description:
+          "Title displayed for the community members table roles column.",
+      }),
+      cell: ({ row: { original: member } }) => roleCell(member.roles, intl),
+    }),
+  ] as ColumnDef<PoolTeamMember>[];
+
+  if (canAddRemoveRoles) {
+    columns = [
+      columnHelper.display({
+        id: "actions",
+        header: intl.formatMessage(tableMessages.actions),
+        cell: ({ row: { original: member } }) =>
+          actionCell(member, pool, canAddRemoveRoles),
+        meta: {
+          hideMobileHeader: true,
+          shrink: true,
+        },
+      }),
+      ...columns,
+    ];
+  }
+
+  const data = useMemo(() => members.filter(notEmpty), [members]);
+
+  return (
+    <>
+      <SEO title={pageTitle} />
+      <Table
+        caption={pageTitle}
+        data={data}
+        columns={columns}
+        sort={{
+          internal: true,
+        }}
+        pagination={{
+          internal: true,
+          total: data.length,
+          pageSizes: [10, 20, 50],
+        }}
+        search={{
+          internal: true,
+          label: intl.formatMessage({
+            defaultMessage: "Search process members",
+            id: "vUtN1U",
+            description:
+              "Label for the process team members table search input",
+          }),
+        }}
+        {...(canAddRemoveRoles && {
+          add: {
+            component: (
+              <AddPoolMembershipDialog pool={pool} members={members} />
+            ),
+          },
+          nullMessage: {
+            description: intl.formatMessage({
+              defaultMessage: 'Use the "Add member" button to get started.',
+              id: "DrR/rp",
+              description: "Instructions for adding a member to a community.",
+            }),
+          },
+        })}
+      />
+    </>
+  );
+};
+
+const ManageAccessPage_PoolQuery = graphql(/* GraphQL */ `
+  query ManageAccessPagePool($poolId: UUID!) {
+    pool(id: $poolId) {
+      ...ManageAccessPagePool
+    }
+  }
+`);
+
+type RouteParams = {
+  poolId: Scalars["ID"]["output"];
+};
+
+const ManageAccessPoolPage = () => {
+  const { poolId } = useRequiredParams<RouteParams>("poolId");
+  const [{ data, fetching, error }] = useQuery({
+    query: ManageAccessPage_PoolQuery,
+    variables: { poolId },
+  });
+
+  const pool = data?.pool;
+
+  return (
+    <AdminContentWrapper>
+      <Pending fetching={fetching} error={error}>
+        {pool ? <ManageAccessPool poolQuery={pool} /> : <ThrowNotFound />}
+      </Pending>
+    </AdminContentWrapper>
+  );
+};
+
+export const Component = () => (
+  <RequireAuth
+    roles={[
+      ROLE_NAME.CommunityAdmin,
```

Comment:
Let's also give PoolOperators and CommunityManagers access to this page. It'll make things less confusing for the transition to the new roles.

Reply:
Sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/11273/commits/5ab10b60378595d9734fcdb1ea91a302710a9ab2

Reply:
Oops https://github.com/GCTC-NTGC/gc-digital-talent/pull/11273/commits/65a716fd8f823625ca2601c9d4db5ab4d12dd8aa